cli/generators/component/templates/component.js

```diff
@@ -0,0 +1,26 @@
+import React, {PropTypes} from 'react';
+
+const propTypes = {
+};
+
+const defaultProps = {
```

Comment:
As we discussed @matiasbeckerle please change it to have the propTypes and defaultProps below. Thanks!